react/Dialog/DialogEffects.ts

```diff
@@ -0,0 +1,102 @@
+import { Theme, useTheme } from '@material-ui/core'
+
+import { getFlagshipMetadata, isFlagshipApp } from 'cozy-device-helper'
+import { FlagshipUI } from 'cozy-intent'
+
+import { useSetFlagshipUI } from '../hooks/useSetFlagshipUi/useSetFlagshipUI'
+
+const makeOnMount = (
+  fullScreen: boolean,
+  hasAnotherDialog: boolean,
+  theme: Theme,
+  sidebar: Element | null,
+  cozyBar: Element | null
+): FlagshipUI =>
+  fullScreen
+    ? {
+        bottomBackground: theme.palette.background.paper,
+        bottomTheme: 'dark',
+        topBackground: theme.palette.background.paper,
+        topTheme: 'dark'
+      }
+    : {
+        ...(hasAnotherDialog
+          ? {}
+          : {
+              bottomBackground:
+                theme.palette.background[sidebar ? 'default' : 'paper']
+            }),
+        bottomTheme: 'light',
+        bottomOverlay: 'rgba(0, 0, 0, 0.5)',
+        topOverlay: 'rgba(0, 0, 0, 0.5)',
+        ...(hasAnotherDialog || !cozyBar
+          ? {}
+          : {
+              topBackground:
+                cozyBar &&
```

Comment:
Do we really need to check `cozyBar` here? It should be protected by the `!cozyBar` condition.

Reply:
You're right, this guard is redundant, removed it.